blueprints/screening_app/domains/prediabetes_screening/answer_age/train.txt

```diff
@@ -0,0 +1,228 @@
+{45|sys_number}
```

Comment:
The only reason I kept them was to avoid having a random entity trigger the opt_in intent. For example, if I were to remove answer_age, a random utterance like "32 años" would trigger the form. Right now they go to the default handler.

I did however remove the implicit yes intents since these seem to be more open.